web/src/stream_settings_components.js

```diff
@@ -45,13 +45,38 @@ export const show_subs_pane = {
         $(".settings").show();
         set_right_panel_title(sub);
     },
-    create_stream() {
+    create_stream(container_name = "configure_channel_settings") {
+        $(".stream_creation_container").hide();
+        if (container_name === "configure_channel_settings") {
+            $("#subscription_overlay .stream-info-title").text(
+                $t({defaultMessage: "Create channel: configure settings"}),
+            );
+        } else {
+            $("#subscription_overlay .stream-info-title").text(
+                $t({defaultMessage: "Create channel: add subscribers"}),
+            );
+        }
+        update_footer_buttons(container_name);
+        $(`.${container_name}`).show();
         $(".nothing-selected, .settings, #stream-creation").hide();
         $("#stream-creation").show();
-        $("#subscription_overlay .stream-info-title").text($t({defaultMessage: "Create channel"}));
     },
 };
 
+export function update_footer_buttons(container_name = "configure_channel_settings") {
```

Comment:
I think there is no need for a default value as the caller will pass container name always.